examples/stochastic_xymodel/stochastic_xymodel.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import division
```

Comment:
Thanks for adding the exmaple :).  Can you add some text at the top explaining what this example does?